server/models/Notification.js

```diff
@@ -132,6 +132,28 @@ export default function(Sequelize, DataTypes) {
       },
     }).then(us => us.map(us => us.UserId));
   };
+  /**
+   * Check if notification with `notificationType`, `CollectiveId` and `UserId` is active.
+   * @param {*} notificationType
+   * @param {*} CollectiveId
+   * @param {*} UserId
+   */
+  Notification.isActive = (notificationType, CollectiveId, UserId) => {
+    debug('isActive', notificationType, CollectiveId, UserId);
+    return models.Notification.findOne({
+      where: {
+        CollectiveId,
+        type: notificationType,
+        UserId,
+      },
+    }).then(notification => {
+      if (notification) {
+        return notification.active;
```

Comment:
Is that properly returned as a boolean by the model?

Reply:
Yeah, I think so, the active field  `sequelize.Type` is set to boolean.

Reply:
Does that mean that we don't send a notification unless the user has an active `Notification`?

We did that at the beginning but then we realized that it was much better to do the opposite.
Always send a notification *unless* the user explicitly asked not to.

Since you are returning true if there is no row, it might still be the case. So I think this is good. Just wanted to make sure the purpose of the `Notifications` table is well understood (the `active` column should in fact always be `false`).

Reply:
@xdamman I think we got it right and are on the same page.